### PR TITLE
Added timetable, slot, days and courses models and API endpoints for both student as well as cr side

### PR DESCRIPTION
**Pull request by architprasar**
@ashutoshc8101 please review the pr

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "cmake.configureOnOpen": true
```
Comment (ashutoshc8101) [inline .vscode/settings.json:2]: Revert changes to this file.

File: api/mess/views.py [hunk 2]
```diff
@@ -85,7 +85,7 @@ def put(self, request, *args, **kwargs):
 
 
 class ComplaintView(APIView):
-    permission_classes = [IsAuthenticated]
+    permission_classes = [AllowAny]
```
Comment (ashutoshc8101) [inline api/mess/views.py:88]: Why `AllowAny` is used here? Complaint should only be created by student.

**Review by ashutoshc8101 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (architprasar) [reply .vscode/settings.json]: done

File: api/mess/views.py (hunk 2 above)
Comment (architprasar) [reply api/mess/views.py]: done

**Comment by architprasar** [comment]
@ashutoshc8101 please review again

**Review by ashutoshc8101 — APPROVED** [review]

**Pull request merged**